transcom/mymove
# B-18194 INT add auth check to weight ticket update

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a871910)

## Summary

Make sure a MilMove user is authorized to modify the data

### How to test

1. Login as a customer with an existing move in the PPM Upload documents stage (or create one and get to that step in the process)
2. Create a weight ticket, update it, and ensure everything works as normal/expected
